pyupgrade.py

```diff
@@ -1124,13 +1193,16 @@ def visit_Name(self, node):  # type: (ast.Name) -> None
         self._visit_simple(node)
 
     def visit_Call(self, node):  # type: (ast.Call) -> None
+        last_frame = self._class_info_stack and self._class_info_stack[-1]
```

Comment:
it's not clear to me why this was moved out, now we're computing this on every invocation instead of just when it matters

Reply:
Line length.  I can noqa it instead or make a sub-function